sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -5,7 +5,16 @@
 ### Breaking Changes
 
 - Removed `Azure::Storage::Files::Shares::PackageVersion`.
-
+- Renamed `GetUserDelegationKeyOptions::startsOn` to `GetUserDelegationKeyOptions::StartsOn`.
+- Removed `ShareClient::ListFilesAndDirectories`.
```

Comment:
Why are we doing this change btw?
```suggestion
- Removed `ShareClient::ListFilesAndDirectories()`.
```

Reply:
To be aligned with other SDK. In all other SDKs, `ListFilesAndDirectories` only exists in directory client, not share client.